build_tools/cmake/iree_cc_binary.cmake

```diff
@@ -224,6 +224,13 @@ function(iree_complete_binary_link_options)
           else()
             list(APPEND _ALWAYS_LINK_DEPS "-WHOLEARCHIVE:${_DEP}")
           endif()
+        elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
```

Comment:
Can you also update the comment above, since it looks like CMake's handling is similar between these two platforms?

Reply:
Done.